src/cljs/athens/db.cljs

```diff
@@ -222,6 +223,12 @@
 (defonce dsdb (d/create-conn schema))
 
 
+(defonce tx-update (r/atom {}))
+
+
+(d/listen! dsdb :tx-update #(reset! tx-update %))
```

Comment:
I'm not sure this is the right approach, because:
- it keeps a toplevel ratom to a tx-report, which will retain in memory the last db state
- it's not clear what the behaviour is when changing databases